jena-shacl/testing/std/AFS

```diff
@@ -0,0 +1,20 @@
+https://github.com/w3c/data-shapes
+https://github.com/w3c/data-shapes/tree/gh-pages/data-shapes-test-suite/tests
+2019-06-09
+  
+Changes:
+# Skip
+In: sparql/pre-binding/manifest.ttl
+
+# Optional $shapesGraph not supported.
+<shapesGraph-001.ttl>
+
+## Various pre-binding cases that should be errors
+## but these are unliekly to be unstable laong term.
```

Comment:
Is this file like a `TODO` list? If not, if used for something, then maybe s/unliekly/unlikely and s/laong/long ?

Reply:
Thanks for catching that. Renamed as README - this file documents the local differences for unsupported features and not rejecting in negative syntax tests.